templates/website/pol-info-survey.html

```diff
@@ -0,0 +1,21 @@
+<?php	
+	
+$values['title'] = "Help us by answering a few quick questions";	
+$values['robots'] = 'noindex, nofollow';	
+	
+template_draw('header', $values);	
+	
+?>
+
+<script type="text/javascript" >
+document.write('<script src="http' + ( ("https:" == document.location.protocol) ? "s" : "") + 
+                '://www.surveygizmo.com/s3/4342879/WTT-Political-Information-Survey?__no_style=true&__output=embedjs&__ref=' +
+                escape(document.location.origin + document.location.pathname) +
+                '&recipient_id=<?=$values['recipient_id']?>&first_use=<?=$values['questionnaire_1_yes'] ? 'true' : 'false'; ?>" type="text/javascript" ></scr' + 'ipt>');
+</script>
+<noscript>
+<a href="https://www.surveygizmo.com/s3/4342879/WTT-Political-Information-Survey?__no_style=true&?jsfallback=true&recipient_id=<?=$values['recipient_id']?>&first_use=<?=$values['questionnaire_1_yes'] ? 'true' : 'false'; ?>">Please click here to answer additional questions.</a>
```

Comment:
`&` needs to be escaped as `&amp;`.

Reply:
This breaks the parameter passing, will PHP then undo it before rendering the html?

Reply:
I don't know what you mean. Ampersands need to be escaped in HTML (that's why they're appearing with a red background above), nothing to do with PHP or parameter parsing?
Also I note an extra `?` before jsfallback that presumably shouldn't be there.

Reply:
I think those are fine because they're dealing with parameters in the URL and aren't being rendered as text.

Reply:
What do you mean by "those"? It says `&?jsfallback=true`, that should be `&jsfallback=true` unless the variable is `?jsfallback` which seems doubtful.
The whole URL should be, in order to be valid HTML, `https://www.surveygizmo.com/s3/4342879/WTT-Political-Information-Survey?__no_style=true&amp;jsfallback=true&amp;recipient_id=<?=$values['recipient_id']?>&amp;first_use=<?=$values['questionnaire_1_yes'] ? 'true' : 'false'; ?>`
I'm not sure what the issue is.

Reply:
That's not the URL that will be visited, because when it is used in the browser, the `&amp;` will be unescaped and the visited URL will contain `&`. But without escaping `&` in URLs, the browser has no way of knowing whether `&nbsp=1` means you meant a non-breaking space or a variable called `nbsp`.

Reply:
With you, that last bit was my issue (testing the url without making a browser render it).